src/main/cpp/t08_chess.cpp

```diff
@@ -24,6 +24,36 @@
 using namespace std;
 
 int t08_chess() {
-    
+    int n, m, num;
+	cin >> n >> m;
+	int arr[n][m];
+	num = 1;
+	for (int i = 0; i < n; i++) {
+			if (i % 2 == 0) { 
+			    for (int j = 0; j < m; j+=2){
+				arr[i][j] = num;
+				num++;
+          }
+        for (int j = 1; j < m; j+=2){
+				arr[i][j] = 0;
+			}
+			}
+			else if ( i % 2 != 0){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Condition 'i%2!=0' is always true](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4731462)